extensions/amp-bind/0.1/amp-state.js

```diff
@@ -43,7 +43,8 @@ export class AmpState extends AMP.BaseElement {
   activate(invocation) {
     const event = invocation.event;
     if (event && event.detail) {
-      this.updateState_(event.detail.response, /* isInit */ false);
+      this.updateState_(event.detail.response,
```

Comment:
Yes, the intention was usage with `amp-form` to merge XHR JSON state into `amp-state`, e.g.
```
<form on="submit-success:myAmpState">
```
So in this case, a state update could mutate `amp-state[src]` and cause another XHR but that can't cause a cycle because forms can't be submitted via a bind mutation.